.changeset/fuzzy-pans-move.md

```diff
@@ -0,0 +1,28 @@
+---
+"@neo4j/cypher-builder": patch
```

Comment:
Overall I'm a big fan of this change, it's an easy improvement of the ergonomics of the library, although, as we discussed, my single argument is that this change is a breaking change.
For clarity this is the breaking change, that I'm referring to:

```ts
const testPredicate = Cypher.or(undefined, undefined);
if (testPredicate) {
    console.log("current behaviour")
} else {
    console.log("previous behaviour")
}
```
We could consider the above an unintended behaviour but up to you really!